### PR TITLE
Update build output

### PR DESCRIPTION
**Pull request by drwpow**
## Changes

We had a placeholder output in build that didn’t print real stats. While this won’t be the final build output, this is just “not made up.“

### `npm run build`

In the new default build, **all pages are hidden** (it was just noisy)

Screen Shot 2021-11-12 at 15 52 08

### `npm run build -- --verbose`

However, you can opt-in to seeing the output in **verbose mode**

Screen Shot 2021-11-12 at 15 52 20

Screen Shot 2021-11-12 at 15 53 17

## Testing

No testing needed for stats display

## Docs

No docs needed

### CODE REVIEW

File: packages/astro/src/core/build/index.ts [hunk 1]
```diff
@@ -232,21 +250,13 @@ class AstroBuilder {
...
     /* eslint-disable no-console */
-    console.log(`${bold(cyan('Done'))}
-Pages (${pageCount} total)
-  ${green(`✔ All pages under ${kb(html.maxSize)}`)}
-JS
-  ${pad('initial load', 50)}${pad(kb(js.entryHTML || 0), 8, 'left')}
-  ${pad('total size', 50)}${pad(kb(js.total), 8, 'left')}
-CSS
-  ${pad('initial load', 50)}${pad('0 kB', 8, 'left')}
-  ${pad('total size', 50)}${pad('0 kB', 8, 'left')}
```
Comment (matthewp) [inline packages/astro/src/core/build/index.ts:247]: haha, i can't believe I didn't notice this while working on the build

**Review by matthewp — APPROVED** [review]
lgtm

**Comment by matthewp** [comment]
If you rebase with `main` this should pass again now.

File: packages/astro/src/core/build/index.ts (hunk 1 above)
Comment (drwpow) [reply packages/astro/src/core/build/index.ts:247]: 😛 haha shows how many people pay attention to build output! /s

**Pull request merged**